doc/api_ref/block_cipher.rst

```diff
@@ -134,6 +22,14 @@ block of plaintext with AES-256 using two different keys.
 .. literalinclude:: /../src/examples/aes.cpp
    :language: cpp
 
+API Overview
+------------
+
+.. container:: toggle
+
+   .. doxygenclass:: Botan::BlockCipher
+      :members: set_key,minimum_keylength,maximum_keylength,encrypt,decrypt,encrypt_n,decrypt_n
```

Comment:
Before it also documented the static `create()` and `create_or_throw()`. I would say we probably want to continue mentioning them.

Reply:
Good point. Done.